src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/index.tsx

```diff
@@ -212,7 +212,7 @@ class IssueRuleEditor extends AsyncView<Props, State> {
     const endpoint = `/projects/${organization.slug}/${project.slug}/rules/${ruleId}`;
 
     if (rule && rule.environment === ALL_ENVIRONMENTS_KEY) {
-      delete rule.environment;
+      delete (rule as any).environment;
```

Comment:
changing the type of `environment ` didn't seem to make sense in this case. Just casted it to any to get it to stop.